src/components/List/List.tsx

```diff
@@ -92,6 +93,7 @@ const generateItems = (
         {...liProps}
       >
         {body}
+        {middot ? " " : null}
```

Comment:
Would be good to have some comment around here to explain that the middot variant in Vanilla relies on whitespace between inline elements.

Reply:
Sure, will add.